common/vpcclient/vpcfilevolume/expand_volume.go

```diff
@@ -45,8 +45,11 @@ func (vs *FileShareService) ExpandVolume(shareID string, volumeTemplate *models.
 	request := vs.client.NewRequest(operation)
 	req := request.PathParameter(shareIDParam, shareID)
 	ctxLogger.Info("Equivalent curl command and payload details", zap.Reflect("URL", req.URL()), zap.Reflect("Payload", volumeTemplate), zap.Reflect("Operation", operation))
-	_, err := req.JSONBody(volumeTemplate).JSONSuccess(&share).JSONError(&apiErr).Invoke()
+	resp, err := req.JSONBody(volumeTemplate).JSONSuccess(&share).JSONError(&apiErr).Invoke()
 	if err != nil {
+		if len(apiErr.Errors) > 0 {
+			apiErr.Errors[0].Status = resp.Status
```

Comment:
what is the use of updating apiErr as you are not using it anywhere, if this is the case then why we can't do it in Invoke method only

Reply:
@arahamad We already discussed on this apiErr is the object mapped to error and filled by API error response. We can discuss again. Invoke method does populate the apierror object but in that we dont get status. So we have to populate it.